src/models/index.js

```diff
@@ -1,84 +1,89 @@
 'use strict';
 
-
-const { DataTypes, Sequelize, DATABASE_URL, sequelizeOption } = require("../config")
-
-const createItemSaleModel = require('./schemas/itemSale.model')
-const createItemRentalModel = require('./schemas/itemRental.model')
-const createOrderModel = require('./schemas/order.model')
-const createCategoryModel = require("./schemas/category.model");
-const createEventModel = require("./schemas/event.model");
+const {
+  DataTypes,
```

Comment:
please disable the prettier and re-push to avoid unnecessary merge conflicts

make the import on a single line